docs/docs/10-reference.md

```diff
@@ -147,7 +176,17 @@ $ snowpack build --clean
 - **`buildOptions.webModulesUrl`** | `string` | Default: `web_modules`
   - Rename your web modules directory.
 
-#### Proxy Options
+#### `config.proxy`
+
+`object` (path: options)
+
+If desired, `"proxy"` is where you configure the proxy behavior of your dev server. Define different paths that should be proxied, and where they should be proxied to.
+
+The short form of a full URL string is enough for general use. For advanced configuration, you can use the object format to set all options supported by [http-proxy](https://github.com/http-party/node-http-proxy).
```

Comment:
Basic structure of this section, proposed:

1. Document the short form URL style
2. Document the long form URL style by pointing to http-proxy documention
3. BUT we need to document the `on()` handlers as well! Those are custom for us, so make it clear that "we support http-proxy BUT extend it with this one special thing"